src/SocketSession.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021, Group5 Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#pragma once
+#include<string>
+#include "SocketUtility.h"
+// Handle the upgrading http request.
```

Comment:
Please add a line after the include, see:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done